README.md

```diff
@@ -14,9 +14,13 @@
     - Start a new terminal session so that the recently edited .bashrc is in effect. Notice (.venv) in the prompt. Also, check for your OpenAI API key using the command `echo $OPENAI_API_KEY`
     - Issue command `make install`
 
-How to run?
-1. Add your OpenAI key in "bot_files/config.py"
-2. Add resumes to the folder "files" or through the UI. (Note: Currently the embeddings are created and loaded on the existing database. 
-3. If you want to recreate the embeddings, Run update_database.py
-4. Run index.py and open localhost to see the UI.
+- How to run the bot
+    - Add your OpenAI key in "bot_files/config.py"
```

Comment:
Why not read the API Key from the env variable?  I had modified the code to use the env variable in the main program.

For GitHub Actions, if required, we can use GitHub Secrets.